packages/react-server-dom-parcel/src/server/ReactFlightDOMServerNode.js

```diff
@@ -88,6 +95,7 @@ type PipeableStream = {
 
 export function renderToPipeableStream(
   model: ReactClientValue,
+
```

Comment:
```suggestion
```
Is that by chance an intentional gap for the not-needed-for-parcel `ClientManifest` to be able to better diff the implementation between the three bundlers, or just leftover copy pasta?